translator/internaldata/resource_to_oc_test.go

```diff
@@ -74,6 +78,36 @@ func TestResourceToOC(t *testing.T) {
 	}
 }
 
+func TestContainerResourceToOC(t *testing.T) {
+	resource := pdata.NewResource()
+	resource.InitEmpty()
+	resource.Attributes().InitFromMap(map[string]pdata.AttributeValue{
+		conventions.AttributeK8sCluster:    pdata.NewAttributeValueString("cluster1"),
+		conventions.AttributeK8sPod:        pdata.NewAttributeValueString("pod1"),
+		conventions.AttributeK8sNamespace:  pdata.NewAttributeValueString("namespace1"),
+		conventions.AttributeContainerName: pdata.NewAttributeValueString("container-name1"),
+		conventions.AttributeCloudAccount:  pdata.NewAttributeValueString("proj1"),
+		conventions.AttributeCloudZone:     pdata.NewAttributeValueString("zone1"),
+	})
+
+	want := &ocresource.Resource{
+		Type: resourcekeys.ContainerType, // Inferred type
```

Comment:
Consider adding another test for the case where `Type` is explicitly set (via `OCAttributeResourceType`) to ensure this is not overridden.

Reply:
Done, at the bottom of the same test.